templates/find-a-partner/index.html

```diff
@@ -7,101 +7,94 @@
 
 {% block content %}
-<div class="row row-hero no-border no-padding-bottom no-margin-bottom">
-    <div class="twelve-col equal-height">
-        <div class="eight-col">
-            <h1>Find an Ubuntu partner</h1>
-            <p class="intro">Canonical&rsquo;s network of Ubuntu partners spans the full range of technology activities. We offer partner programmes for software vendors, hardware OEMs, mobile carriers, channel partners, retailers and public cloud providers. </p>
-        </div>
-        <div class="four-col last-col align-center align-vertically">
-            <img class="priority-0" src="{{ STATIC_URL }}img/partners-search.svg" width="180" alt="Search pictogram">
-        </div>
-    </div>
-</div><!-- /row -->
 
-<div class="row no-border">
-    <div class="search-panel three-col">
-        <div class="twelve-col">
-            <form>
-                <input type="text" name="search" id="search" class="search">
-            </form>
-        </div>
-        <div class="twelve-col">
-            <div class="tabbed-menu" style="display: none;">
-                <ul>
-                    {% for filter in filters %}
-                    <li>
-                    <a href="#{{ filter.name|slugify }}" class="active">{{ filter.name }}</a>
-                    </li>
-                    {% endfor %}
-                </ul>
-            </div>
-            {% for filter in filters %}
-            <div id="{{ filter.name|slugify }}" class="tabbed-content">
-                <a href="#{{ filter.name|slugify }}" class="accordion-button slideless">{{ filter.name|lower|capfirst }}</a>
-                <ul class="no-bullets">
-                    {% for item in filter.items %}
-                    <li>
-                        <input type="checkbox" name="{{ filter.name|slugify }}-{{ item|slugify }}" id="{{ filter.name|slugify }}-{{ item|slugify }}">
-                        <label for="{{ filter.name|slugify }}-{{ item|slugify }}">{{ item }}</label>
-                    </li>
-                    {% endfor %}
-                </ul>
-            </div>
-            {% endfor %}
-        </div>
+<section class="p-strip--image is-dark" style="background-image: url('https://assets.ubuntu.com/v1/775cc62b-vanilla-grad-background.png'); background-position: 75% 50%;">
+  <div class="row">
+    <h1 class="u-no-margin--bottom">Find an Ubuntu partner</h1>
+    <p class="intro">Canonical&rsquo;s network of Ubuntu partners spans the full range of technology activities. We offer partner programmes for software vendors, hardware OEMs, mobile carriers, channel partners, retailers and public cloud providers. </p>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-4 p-card">
+      <h3>Search</h3>
+      <form>
+        <input type="text" name="search" id="search" class="p-find-a-partner__search-input">
```

Comment:
Should there be a submit button here so that search works without JS?

Reply:
@anthonydillon the search doesn't work without JS

Reply:
Could you make it work without js? Or at least create an issue to follow up with that functionality.

Reply:
@anthonydillon writing a backend search is well outside of the scope of the Vanilla migration of the site. I'll create an issue.